source/release-notes/1.4.txt

```diff
@@ -27,7 +25,57 @@ Starting in 1.4.0, you can set the workload level that
 destination clusters. For more information, see the
 :option:`--loadLevel` option.
 
+Improved Resiliency
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` 1.4.0 improves :term:`oplog` handling. The ``oplog``
+handling improvement adds resiliency to long running migrations.
+
+Faster Processing for Sharded Clusters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Version 1.4.0 improves performance for replica sets and single-shard
+clusters. Multi-shard clusters have even better speed improvements.
```

Comment:
Since we don't quantify speed, should we combine and say replica sets and sharded clusters have improved performance? 

Is a particular stage (collection copy or CEA) sped up? @davemungo

Reply:
(The optimization work focuses on CEA.)